fastlane/lanes/test.rb

```diff
@@ -0,0 +1,41 @@
+GOOGLE_FIREBASE_SECRETS_PATH = File.join(Dir.home, '.configure', 'wordpress-android', 'secrets', 'firebase.secrets.json')
+
+platform :android do
+  #####################################################################################
+  # build_and_instrumented_test
+  # -----------------------------------------------------------------------------------
+  # Run instrumented tests in Google Firebase Test Lab
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_instrumented_test
+  #
+  #####################################################################################
+  desc "Build the application and instrumented tests, then run the tests in Firebase Test Lab"
+  lane :build_and_instrumented_test do | options |
```

Comment:
Should this say 

```suggestion
  lane :build_and_run_instrumented_test do | options |
```

?🤔

Reply:
\+ 💯  on this: when I saw the name of that lane after getting up-to-date with recent changes after my AFK, I was like "this doesn't read right, this isn't proper English 😅" which made this lane name a bit confusing and uneasy to read 🙃 

@pachlava if you're still working on a subsequent PR to address the leftovers of this one (e.g. making the tests that were disabled here work again), maybe you can take the occasion to do a quick rename of the lane while at it?

Reply:
Sure @AliSoftware ! Thanks for confirming that this is the case. As a non-native speaker, I wasn't sure if this is a missed word, or the case of usage of `instrumented_test` as a verb, like in "They instrumented tested this yesterday" 🤔 😄